CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-01-08
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 has finished rolling out in all non-sovereign regions.
+* AKS begins pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Features
+  * Starting with Kubernetes 1.26:
+    * HostProcess Containers will be GA
+    * Azure NPM will now enforce ingress network policies for traffic sent via LoadBalancer IP's. The previous behavior NPM would only enforce pod to pod traffic. To avoid a breaking change, verify that you have CIDR block allow rules for your external traffic.
+    * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+  * [Dynamic allocation of IPs for Windows]( https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni#:~:text=AKS%20cluster%20creation%3A-,Dynamic%20allocation%20of%20IPs%20and%20enhanced%20subnet%20support,-A%20drawback%20with) is now generally available
+* Preview Features
+  * [IP based Load Balancer](https://learn.microsoft.com/en-us/azure/aks/load-balancer-standard#:~:text=Change%20the%20inbound%20pool%20type%20(PREVIEW)) is now available in preview
+  * [Node public IP tags](https://learn.microsoft.com/en-us/azure/aks/use-node-public-ips) is now available in preview
+  * [Host port NSG control](https://learn.microsoft.com/en-us/azure/aks/use-node-public-ips) is now available in preview
+  * [Host ASG control](https://learn.microsoft.com/en-us/azure/aks/use-node-public-ips) is now available in preview
+  * [Azure CNI Overlay for Windows](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay) is now available in preview
+* Behavior Changes
+  * The OSM addon's osm-injector pod's autoscaler will no longer scale on memory, only on CPU. The osm-controller pod's HPA resource will be reconciled in EnsureExists mode to allow you to edit the resource.
+* Bug Fixes
+  * A bug regarding Kubernetes version 1.25 and the tigera operator has been [fixed](https://github.com/Azure/AKS/issues/3394). If your cluster is already running on v1.25.4, please create a new cluster or file a support ticket with AKS for any further help.
+* Component Updates
+  * [CIS Kubernetes v1.24 Benchmark](https://learn.microsoft.com/en-us/azure/aks/cis-kubernetes) has been published which covers AKS 1.21.x through AKS 1.24.x
+  * KEDA add-on for AKS has been upgraded to v2.9
```

Comment:
can we link to docs and release notes of v2.9 so that changelog is referenced

Reply:
https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v290 Should be this link, right?